### PR TITLE
[DOC] Add workflowIds parameter

### PR DESCRIPTION
**Pull request by vagimeli**
### Description
Adds workflowIds setting

### Issues Resolved
Fixes https://github.com/opensearch-project/documentation-website/issues/5421

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by vagimeli** [comment]
@eirsep Please tech review and approve the parameter setting changes made in this PR. We are closing a content gap. It should be minimal time effort. Thank you :)

**Comment by vagimeli** [comment]
> @eirsep

Hi @eirsep: Please provide technical review so that we can publish the updated documentation. Thank you.

**Comment by vagimeli** [comment]
@hdhalter I'm blocked on this PR because I'm not getting a tech review. I've requested a review using the AWS OpenSearch and OpenSearch community Slack channels. No response after a week. Who is the SDM?

**Comment by vagimeli** [comment]
@hdhalter Tech review is only needed for line 1346. I've requested tech review via OpenSearch community Slack channel and AWS OpenSearch Slack channel. No response to date. Who is the SDM?

**Comment by hdhalter** [comment]
> @hdhalter I'm blocked on this PR because I'm not getting a tech review. I've requested a review using the AWS OpenSearch and OpenSearch community Slack channels. No response after a week. Who is the SDM?

I'm checking with @getsaurabh02.

**Comment by getsaurabh02** [comment]
Adding @praveensameneni to take a look.

File: _observing-your-data/alerting/api.md [hunk 1]
```diff
@@ -1350,15 +1334,16 @@ The following table lists the available path parameters. All path parameters are
...
+| `sortString` | String | Defines how to sort the results. Default is `monitor_name.keyword`.
+| `sortOrder` | String | Defines the results order. Options are `asc` or `desc`. Default is `asc`.
+| `missing` | String | Specifies whether to include missing data in the response.
+| `size` | String | Defines the size of the request to be returned. Default is `20`.
+| `startIndex` | String | Defines the index to start from. Used for paginating results. Default is `0`.
+| `searchString` | String | Defines the search string to use for searching a specific alert. Default is an empty string.
+| `severityLevel` | String | Defines the severity level to filter for. Default is `ALL`.
+| `alertState` | String | Defines the alert state to filter for. Default is `ALL`.
 | `monitorId` | String | Filters by monitor ID.
+| `workflowIds` | String | Allows for monitoring the status of multiple chained alerts within a single dashboard. Available in OpenSearch 2.9 or later.
```
Comment (eirsep) [inline _observing-your-data/alerting/api.md:1346]: `Allows for monitoring the status of chained alerts from multiple workflows within a single dashboard. Available in OpenSearch 2.9 or later.`

**Review by eirsep — CHANGES_REQUESTED** [review]
suggested a re-wording for the param description

File: _observing-your-data/alerting/api.md [hunk 2]
```diff
@@ -1350,15 +1334,16 @@ The following table lists the available path parameters. All path parameters are
...
-| `sortString` | String | Determines how to sort the results. Defaults to `monitor_name.keyword`.
-| `sortOrder` | String | Determines the order of the results. Options are `asc` or `desc`. Defaults to `asc`.
-| `missing` | String | Optional.
-| `size` | String | Determines the size of the request to be returned. Defaults to `20`.
-| `startIndex` | String | The index to start from. Used for paginating results. Defaults to `0`.
-| `searchString` | String | A search string used to look for a specific alert. Defaults to an empty string.
-| `severityLevel` | String | The severity level to filter for. Defaults to `ALL`.
-| `alertState` | String | The alert state to filter for. Defaults to `ALL`.
+| `sortString` | String | Defines how to sort the results. Default is `monitor_name.keyword`.
+| `sortOrder` | String | Defines the results order. Options are `asc` or `desc`. Default is `asc`.
```
Comment (natebower) [inline _observing-your-data/alerting/api.md:1338]:
```suggestion
| `sortOrder` | String | Defines the order of results. Options are `asc` or `desc`. Default is `asc`.
```

**Review by natebower — APPROVED** [review]
@vagimeli Just one minor change. Otherwise, LGTM!

**Pull request merged**